src/applications/personalization/profile/components/direct-deposit/BankInfoCNP.jsx

```diff
@@ -51,9 +52,31 @@ export const BankInfoCNP = ({
   const isSavingBankInfo = directDepositUiState.isSaving;
   const saveError = directDepositUiState.responseError;
 
-  const { accountNumber, accountType, routingNumber } = formData;
-  const isEmptyForm = !accountNumber && !accountType && !routingNumber;
+  const bankInfoUpdatedAlertSettings = {
```

Comment:
I had to add this stuff to make the Cypress test work nicely. It's how the new Direct Deposit is set up.